src/bin/leftwm-worker.rs

```diff
@@ -24,10 +24,10 @@ fn main() {
     }
 
     let result = panic::catch_unwind(|| {
-        let mut manager = Box::new(Manager::default());
```

Comment:
It there a reason why it was boxed?

Reply:
I think this was needed for the layouts way back in version 0.0.1. You should be able to remove it.